trino-api/internal/app/handler/handler_test.go

```diff
@@ -0,0 +1 @@
+package handler
```

Comment:
atleasts unit test for core functionality of parsing requests should be there IMO

Reply:
i'm facing issue in validating the test cases with sql.Rows interface, bit stuck there